src/checkResultView.ts

```diff
@@ -103,11 +103,13 @@ function ensurePanelBody(extContext: vscode.ExtensionContext) {
     const resourcesDiskPath = vscode.Uri.file(
         path.join(extContext.extensionPath, 'resources')
     );
-    const resourcesPath = resourcesDiskPath.with({ scheme: 'vscode-resource' });
+    const resourcesPath = viewPanel.webview.asWebviewUri(resourcesDiskPath);
     if (!viewHtml) {
         viewHtml = fs.readFileSync(path.join(resourcesPath.fsPath, 'check-result-view.html'), 'utf8');
```

Comment:
```suggestion
        viewHtml = fs.readFileSync(path.join(path.join(extContext.extensionPath, 'resources', 'check-result-view.html')), 'utf8');
```

please see for a reasoning: https://github.com/eclipse-theia/theia/issues/6578#issuecomment-555945435

Reply:
👍